tools/storybook-utils/src/getStoriesPattern.ts

```diff
@@ -0,0 +1,25 @@
+import { getLGConfig } from '@lg-tools/meta';
+
+/**
+ * Returns a glob to match all story files,
+ * based on the scopes and directories defined in
+ * the package.json "lg" config
+ */
+export const getStoriesPattern = (): string => {
+  // TODO: We may want to use pnpm-workspace config here,
+  // since we could have story files in other non-package directories
```

Comment:
What's an example of this?

Reply:
https://github.com/mongodb/leafygreen-ui/blob/main/stories/Introduction.stories.mdx